UseCase.md

```diff
@@ -1,64 +1,82 @@
 
 ### Physical Store
 
-For brick and mortar stores, BTCPay has a [web-based Point of Sale](https://mainnet.demo.btcpayserver.org/apps/87kj5yKay8mB4UUZcJhZH5TqDKMD3CznjwLjiu1oYZXe/pos) (POS) App which can be customized. Similarly to the online store, your customer is presented with an invoice which needs to be paid. You can create a watch-only wallet on your phone to be notified of the payments through the POS, without the need of any additional software. There are plans on developing the POS mobile app which would make the process even smoother.
+For brick and mortar stores, BTCPay has a [web-based Point of Sale](https://mainnet.demo.btcpayserver.org/apps/87kj5yKay8mB4UUZcJhZH5TqDKMD3CznjwLjiu1oYZXe/pos) (POS) App which can be customized. Similarly to the online store, your customer is presented with an invoice that he can pay on the spot. You can create a watch-only wallet on your phone to be notified of the payments through the POS, without the need of any additional software. There are plans to develop the POS mobile app which would make the process even smoother.
+
+### Lightning Network payments
+
+No matter the kind of business you run, BTCPay offers a very easy way to get started on the Lightning Network. You can use and experiment with this innovative second-layer solution build on top of Bitcoin by [following this guide](LightningNetwork.md). Both merchants and customers can use BTCPay to receive or make payments off-chain with instant confirmations and neglectable network fees.
 
 ## Charities and Content Creators (Donations)
 
-Charities, non-profits, content creators and other organization that want to accept cryptocurrency donations in a more private way than the traditional single-address can use the [Pay Button](https://docs.btcpayserver.org/btcpay-basics/gettingstarted#creating-the-pay-button) or the [POS app](https://github.com/btcpayserver/btcpayserver-doc/blob/master/GettingStarted.md#creating-the-point-of-sale-app).
+Charities, non-profits, content creators and other organizations that want to accept cryptocurrencies donations in a more private way than the traditional single-address can use the [Pay Button](https://docs.btcpayserver.org/btcpay-basics/gettingstarted#creating-the-pay-button) or the [POS app](https://github.com/btcpayserver/btcpayserver-doc/blob/master/GettingStarted.md#creating-the-point-of-sale-app).
 
 Benefits of using BTCPay for accepting donations:
 
 * Saving money (no fees, no subscriptions)
-* Payments go directly to your wallet (No third-party)
-* Enhanced privacy for you and donors (no address re-use, no IP leaks to third parties)
+* Cutting off the middle-man (Payments go directly to their wallet)
```

Comment:
> to their wallet
Whose wallet?

Reply:
I hesitated between second and third person and switched many times, forgot to change this one. 

But there is another part where this is a real issue, it is the "local payment processor" part. I just realised I first talk from the perspective of the server admin that becomes payment processor for others, and then list benefits for other people to use a third party hosting. This is really not consistent...

I'm mentioning it because I think I could cut it off from there, but add a sentence in the first "merchants" part just to mention that BTCPay benefits are real even if you use third party hosting. 

I'll do a version like that so you can see what I mean

Reply:
I actually think you handled that quite well in the new version.